### ISSUE TITLE
Use in a VPS

### ISSUE THREAD

**Issue by LennonPereira** [kakaroto/SWProxy#195]
it is possible to run this proxy on a VPS

**Comment by kakaroto** [comment]
Sure, why not ?

**Comment by LennonPereira** [comment]
I have a VPS windows in amazon, but I'm not getting setup.

**Comment by LennonPereira** [comment]
the IP that SW Proxy recognizes inside the VPS is different from the IP that I use for remote connection, I believe this is with some problem.

**Comment by kakaroto** [comment]
Yeah, it might be the problem. Did you try the IP it gives you in SWProxy ? That seems to be the external IP address visible to the internet. Anyways, you could always just hardcode the IP address you want in here : https://github.com/kakaroto/SWProxy/blob/master/SWProxy.py#L83
```
def get_external_ip():
    my_ip = [[(s.connect(('8.8.8.8', 80)), s.getsockname()[0], s.close()) for s in [socket.socket(socket.AF_INET, socket.SOCK_DGRAM)]][0][1]][0]
    return my_ip
```

That should force it to listen on that IP address and if that was your issue, it would fix it. Replace it with something like : 
```
def get_external_ip():
    my_ip = "123.45.67.89"
    return my_ip
```

**Comment by LennonPereira** [comment]
kakaroto I do not develop in Python just in C #, so I have not tried changing the font. will an update be possible where the IP compo is open for typing?

**Comment by kakaroto** [comment]
I don't maintain the app anymore and I don't know if it's a simple fix or
not. I also just don't have time to do that, even if I wanted to. I already
gave you the solution. Just edit the file and set the IP address there.
It's not complicated.

**Comment by LennonPereira** [comment]
I'm seeing some video of how to compile Python

**Comment by kakaroto** [comment]
It's not compiled, it's interpreted, you modify the .py file and run it
again, that's it.
If you use Windows then I'm not sure how that works since it builds it into
a single executable or something.
